deps/wazuh_testing/wazuh_testing/system/host_manager.py

```diff
@@ -0,0 +1,472 @@
+        """Get the ansible facts of the specified host.
+
+        Args:
+            host (str): Hostname
+        Returns:
+            str: OS of the host
+        """
+        testinfra_host = self.get_host(host)
+        return testinfra_host.ansible("setup")
+
+    def collect_host_os(self, host):
+        """Get the OS of the specified host.
+
+        Args:
+            host (str): Hostname
+        Returns:
+            str: OS of the host
+        """
+        ansible_facts = self.get_host_ansible_facts(host)
+        return (ansible_facts['ansible_facts']['ansible_distribution'],
+                ansible_facts['ansible_facts']['ansible_distribution_major_version'],
+                ansible_facts['ansible_facts']['ansible_distribution_version'])
+
+    def collect_host_ips(self, host):
+        """Get the host IPs
+
+        Args:
+            host (str): Hostname
+        Returns:
+            dict: IPs of the host (ipv4 and ipv6)
+        """
+        ansible_facts = self.get_host_ansible_facts(host)
+        return {'ipv4': ansible_facts['ansible_facts']['ansible_all_ipv4_addresses'],
+                'ipv6': ansible_facts['ansible_facts']['ansible_all_ipv6_addresses']}
+
+    def collect_host_interfaces(self, host):
+        """Get the interfaces of the specified host.
+
+        Args:
+            host (str): Hostname
+        Returns:
+            dict: Interfaces of the host
+        """
+        ansible_facts = self.get_host_ansible_facts(host)
+        return ansible_facts['ansible_facts']['ansible_interfaces']
+
+    def check_connection(self, host, windows=False):
+        """Check if the host is reachable.
+
+        Args:
+            host (str): Hostname
+            windows (bool): Use windows command
+        Returns:
+            bool: True if the host is reachable, False otherwise
+        """
+        testinfra_host = self.get_host(host)
+        ansible_command = 'ping' if not windows else 'win_ping'
+
+        return testinfra_host.ansible(ansible_command, check=False)['ping'] == 'pong'
+
+    def copy_file(self, host, src_path, dest_path, remote_src=False, become=False, windows=False, ignore_errors=False):
+        """Move from src_path to the desired location dest_path for the specified host.
+
+        Args:
+            host (str): Hostname
+            src_path (str): Source path
+            dest_path (str): Destination path
+            remote_src (bool): If True, the file is assumed to live on the remote machine, not the controller.
+            become (bool): Use sudo
+            windows (bool): Use windows command
+            ignore_errors (bool): Ignore errors
+        Returns:
+            dict: Result of the command execution
+        Raises:
+            Exception: If the command execution fails
+        """
+        testinfra_host = self.get_host(host)
+        ansible_command = 'copy' if not windows else 'win_copy'
+        remote_source = 'yes' if remote_src else 'no'
```

Comment:
Unused variable, maybe you have to use this variable in `command_parameters` instead of `remote_src`

Reply:
Done db5478b